atlasdb-config/src/main/java/com/palantir/atlasdb/factory/PersistentStoragePathSanitizer.java

```diff
@@ -24,22 +24,30 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
-import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import com.google.common.base.MoreObjects;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.annotations.VisibleForTesting;
 import com.palantir.logsafe.Preconditions;
 import com.palantir.logsafe.SafeArg;
 
-public final class PersistentStorageFactories {
-    private static final Pattern UUID_PATTERN = Pattern.compile(
-            "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$");
+public final class PersistentStoragePathSanitizer {
+    private static final Logger log = LoggerFactory.getLogger(PersistentStoragePathSanitizer.class);
+    private static final PersistentStoragePathSanitizer INSTANCE = new PersistentStoragePathSanitizer();
+    @VisibleForTesting
+    static final String MAGIC_SUFFIX = "atlasdb-persistent-storage";
```

Comment:
I'd shove this into `AtlasdbConstants`

Reply:
in @felixdesouza we believe, so still a magic but now it is private